litgpt/scripts/convert_hf_checkpoint.py

```diff
@@ -321,14 +357,14 @@ def convert_hf_checkpoint(
 
     if "falcon" in model_name:
         copy_fn = partial(copy_weights_falcon, model_name)
-    elif config.mlp_class_name in ("LLaMAMLP", "GemmaMLP", "LLaMAMoE"):
+    elif model_name.startswith(("phi", "Phi")):
```

Comment:
You could also do a `model_name.lower().startswith("phi")`, but not sure that one is better than the other. Looks fine to me as is.

Reply:
Thanks for the update!